subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ResolutionResultApiIntegrationTest.groovy

```diff
@@ -738,4 +738,67 @@ testRuntimeClasspath
             }
 """
     }
+
+    def "resolution result does not realize artifact tasks"() {
+        settingsFile << "include 'producer'"
+        file("producer/build.gradle") << """
+            plugins {
+                id("base")
+            }
+
+            def fooTask = tasks.register("foo", Zip) {
+                throw new RuntimeException("Realized artifact task")
+            }
+
+            configurations {
+                consumable("conf") {
+                    outgoing {
+                        artifact(fooTask)
+                    }
+                    attributes {
+                        attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category.class, "cat"))
+                    }
+                }
+            }
+        """
+
+        buildFile << """
+            configurations {
+                dependencyScope("deps")
+                resolvable("conf") {
+                    extendsFrom(deps)
```

Comment:
This does look odd. Changed the resolvable configuration to use the legacy factory. 

IMO long run, we should dedicate our efforts to the new resolution API instead of building on Configuration